qCC/ccCommandLineCommands.cpp

```diff
@@ -1842,6 +1843,49 @@ bool CommandSFConvertToRGB::process(ccCommandLineInterface &cmd)
 	return true;
 }
 
+CommandRGBConvertToSF::CommandRGBConvertToSF()
+	: ccCommandLineInterface::Command(QObject::tr("RGB convert to SF"), COMMAND_RGB_CONVERT_TO_SF)
+{}
+
+bool CommandRGBConvertToSF::process(ccCommandLineInterface &cmd)
+{
+	cmd.print(QObject::tr("[RGB CONVERT TO SF]"));
+	if (cmd.clouds().empty())
+	{
+		return cmd.error(QObject::tr("No point cloud on which to convert RGB to SF! (be sure to open one with \"-%1 [cloud filename]\" before \"-%2\")").arg(COMMAND_OPEN, COMMAND_RGB_CONVERT_TO_SF));
+	}
+
+	for (CLCloudDesc& thisCloudDesc : cmd.clouds())
+	{
+		ccPointCloud* cloud = thisCloudDesc.pc;
+
+		if (!cloud->hasColors())
+		{
+			cmd.warning(QObject::tr("Cloud %1 has no colors").arg(cloud->getName()));
+			continue;
+		}
+
+		ccHObject::Container container;
+		container.push_back(cloud);
+		if (!ccEntityAction::sfFromColors(container, exportR=False, exportG=False, exportB=False, exportAlpha=False, exportC=True)) //beta version, only composite
```

Comment:
I guess it doesn't compile? 😅 

you can either remove the 'exportR=', etc. or add least comment them:
/*exportR=*/false, /*exportG=*/false, etc.

'and mind the 'false' instead of 'False'